pre-docs/inventory_version2.md

```diff
@@ -96,6 +100,48 @@ groups:
 
 Make sure you have [configured PuppetDB](./bolt_connect_puppetdb.md)
 
+##### Fact Lookups
+
+The PuppetDB plugin also accepts fact lookups as values for `name`, `uri`,
+and values in the `config` hash which will be resolved to the facts for the
+nodes in the PuppetDB query. The fact lookups accept [PQL dot
```

Comment:
If more specific config is required the puppetdb plugin can be used to lookup configuration values for the `name`, `uri`, and `config` inventory options for each target. The fact lookup values can be either `certname` to reference the certaname of the target or a pql dot notation `facts` string such as `facts.os.family` to reference fact value.(link to dot notation docs)

note: If the `name` or `uri` values are set to a lookup the puppetdb inventory plugin will not set the `uri` to the certname of the target.

I'm not sure exactly what to do here but:
* dot notation is required for both structured and unstructured facts.
* I think listing the top level things that can be referenced `certname` and `facts` is clearer than  listing the cofig keys twice with each top level reference.
* I think it needs to mention the magic around when `uri` is set or unset.

Reply:
I tried linking back up to the other docs that explain it.